src/workerd/tests/bench-fast-api.c++

```diff
@@ -42,7 +42,7 @@ struct FastMethodFixture: public benchmark::Fixture {
 BENCHMARK_F(FastMethodFixture, SlowAPI)(benchmark::State& state) {
   FastMethodIsolate isolate(system, kj::heap<jsg::IsolateObserver>(), {});
   auto code =
-      "var result = 0; for (let i = 0; i < 100000; i++) { result += slowAdd(2, 3); } result"_kj;
+      "var result = 0; for (let i = 0; i < 500000; i++) { result += slowAdd(2, 3); } result"_kj;
```

Comment:
PR description says "double" ... this is obviously more than double. Not sure if this change is incorrect or if the PR title is incorrect.